source/Octopus.Tentacle/Commands/RegisterKubernetesClusterCommand.cs

```diff
@@ -9,6 +9,7 @@
 
 namespace Octopus.Tentacle.Commands
 {
+    [Obsolete("This class is being deprecated in favour of RegisterKubernetesDeploymentTargetCommand")]
```

Comment:
If possible make a note around when this could be deleted or of it needs to stay around forever.